### PR TITLE
changes in SQ for Developer refresh CE task in Dev Cockpit

### PR DESCRIPTION
**Pull request by sns-seb**

### CODE REVIEW

File: sonar-db/src/main/java/org/sonar/db/measure/MeasureMapper.java [hunk 1]
```diff
@@ -32,6 +32,10 @@
 
   List<MeasureDto> selectBySnapshotAndMetricKeys(@Param("snapshotId") long snapshotId, @Param("metricKeys") List<String> metricKeys);
 
+  List<MeasureDto> selectByDeveloperForSnapshotAndMetrics(@Param("developerId") long developerId, @Param("snapshotId") long snapshotId, @Param("metricIds") List<Integer> metricIds);
```
Comment (SonarTech) [inline sonar-db/src/main/java/org/sonar/db/measure/MeasureMapper.java]: MINOR Split this 181 characters long line (which is greater than 180 authorized). [rule](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS00103)

**Comment by SonarTech** [comment]
SonarQube analysis reported 1 issue:
* MINOR 1 minor

Watch the comments in this conversation to review them.

**Comment by julienlancelot** [comment]
I would remove the @Nullable in MeasureDto setId() and setComponentId() then LGTM

**Comment by sns-seb** [comment]
> I would remove the @Nullable in MeasureDto setId() and setComponentId() then LGTM

I've simply removed the commit

**Pull request merged**